Doc/c-api/veryhigh.rst

```diff
@@ -24,22 +24,10 @@ use different libraries, so care should be taken that :c:type:`FILE\*` parameter
 are only passed to these functions if it is certain that they were created by
 the same library that the Python runtime is using.
 
+.. _note:
 
-.. c:function:: int Py_Main(int argc, wchar_t **argv)
-
-   The main program for the standard interpreter.  This is made available for
-   programs which embed Python.  The *argc* and *argv* parameters should be
-   prepared exactly as those which are passed to a C program's :c:func:`main`
-   function (converted to wchar_t according to the user's locale).  It is
-   important to note that the argument list may be modified (but the contents of
-   the strings pointed to by the argument list are not). The return value will
-   be ``0`` if the interpreter exits normally (i.e., without an exception),
-   ``1`` if the interpreter exits due to an exception, or ``2`` if the parameter
-   list does not represent a valid Python command line.
-
-   Note that if an otherwise unhandled :exc:`SystemExit` is raised, this
-   function will not return ``1``, but exit the process, as long as
-   ``Py_InspectFlag`` is not set.
+   The documentation for :c:func:`Py_Main` that previously appeared in this
+   section has been moved to :ref:`initialization`.
```

Comment:
I don't think that this note is useful.

Reply:
My rationale for including it is that any old links to https://docs.python.org/3/c-api/veryhigh.html#c.Py_Main will still end up here.

OTOH, https://www.google.com/search?q=link%3Ahttps%3A%2F%2Fdocs.python.org%2F3%2Fc-api%2Fveryhigh.html%23c.Py_Main strongly suggests that direct links to that are rather rare.

Reply:
Removed